test/lib/db.spec.ts

```
@@ -0,0 +1,193 @@
+        // Make client.end throw an error
+        const shutdownError = new Error("Failed to close connection");
+        mockEnd.mockRejectedValueOnce(shutdownError);
+
+        // Import the module
+        await import("~/src/lib/db");
+
+        const shutdownCalls = mockOn.mock.calls.find(
+            (call) => call[0] === "SIGINT",
+        );
+
+        let shutdownHandler;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)